<ui version="4.0" >
 <author>John Eric Martin &lt;john.eric.martin@gmail.com></author>
 <class>SkillList</class>
 <widget class="QWidget" name="SkillList" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>284</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="filterLabel" >
       <property name="text" >
        <string>Filter:</string>
       </property>
       <property name="buddy" >
        <cstring>filterCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="filterCombo" />
     </item>
     <item row="1" column="1" >
      <widget class="SearchEdit" name="searchEdit" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="searchLabel" >
       <property name="text" >
        <string>Search:</string>
       </property>
       <property name="buddy" >
        <cstring>searchEdit</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="skillsList" >
     <property name="iconSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="refreshButton" >
       <property name="text" >
        <string>&amp;Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton" >
       <property name="text" >
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchEdit</class>
   <extends>QLineEdit</extends>
   <header>SearchEdit.h</header>
  </customwidget>
 </customwidgets>
 <includes/>
 <resources/>
 <connections/>
</ui>
